content/guides/user-guides/nbp/csi-pnp/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Container Storage Interface(CSI) Plug-N-Play provided by SODA
+menuTitle: CSI Plug-N-Play
+description: "User guide for Soda CSI Plug-N-Play feature"
+weight: 10
+disableToc: false
+tags: ["user guide", "csi", "plug-n-play"] 
+---
+
+Soda CSI Plug-N-Play allows users to provision the storage in K8s using SODA way and grab the Data Management capabilities offered by SODA.
+SODA CSI PnP let's user to define the profile in SODA and use the profile in the StorageClass of K8s to provision the storage 
+dynamically. It allows user to switch between different heterogeneous CSI drivers via Profile.
+    
+Before we begin we need to create the Profile in SODA using the below command
+```go
+osdsctl profile create
+{
+  "name": "openebs-profile-soda",
+  "storageType": "block",
+  "description": "string",
+  "provisioningProperties": {
+    "dataStorage": {
+      "recoveryTimeObjective": 10,
+      "provisioningPolicy": "Thick",
+      "compression": false,
+      "deduplication": false,
+      "characterCodeSet": "ASCII",
+      "maxFileNameLengthBytes": 255,
+      "storageAccessCapability": [
+        "Read"
+      ]
+    },
+    "ioConnectivity": {
+      "accessProtocol": "iscsi",
+      "maxIOPS": 150,
+      "minIOPS": 50,
+      "maxBWS": 5,
+      "minBWS": 1,
+      "latency": 1
+    }
+  },
+  "replicationProperties": {},
+  "snapshotProperties": {},
+  "dataProtectionProperties": {},
+  "customProperties": {
+    "driver": "local.csi.openebs.io"            ## YOU NEED TO UPDATE THE DRIVER WITH VENDOR DRIVER NAME
+  }
+}
+```
+
+Currently SODA CSI PnP has been tested for the below CSI drivers, this is not the exhaustive list but it will keep growing as other vendors verifies the soda-csi-provisioner.  
```

Comment:
We have a support matrix section now. You can probably add this over there.

Reply:
makes sense, moved it to Support Matrix

Reply:
now using "Soda" in all the places

Reply:
The naming convention we follow is SODA (all uppercase).
Sorry for not being clear earlier. (I re-read my earlier comment and see how it can be misinterpreted.)